Automattic/jetpack
# Super Cache: Make sure plugin links is an array before using it

It was reported that adding the Settings link to the plugins links on the Plugins page caused an error for a user because $links was null. This patch checks to make sure that $links is actually an array before using it.

Fixes [#37600](https://github.com/Automattic/jetpack/issues/37600)

## Proposed changes:
* Check that $links is an array

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
Apply PR
You may need to adjust the path to wp-cache.php in the PR to match how you have WPSC installed. It may be installed in wp-super-cache-dev or super-cache.
Load Plugins page and note the links below the WP Super Cache title.

I can't reproduce this error, but this one line fix should fix it.
